src/pages/Checkout/CheckoutPage.vue

```diff
@@ -992,9 +992,6 @@ export default {
 						}
 					});
 				});
-				Promise.all(loans).then(() => {
```

Comment:
Did you verify that the cache will update the the loan data after the mutation?

The end result should be that the Team Selection box for each Loan in the basket shows the newly joined team as selected.

Reply:
Do you mean this?
<img width="589" alt="Screen Shot 2022-08-19 at 18 39 19" src="https://user-images.githubusercontent.com/94026278/185719745-85ee3637-7ae6-43eb-8a6f-4101ddab2673.png">

Reply:
Correct. Just wanted to make sure the new team was selected after joining. You may be to remove your self from that team now to test this again...

Reply:
Nice, I tested a couple of times adding and removing myself from that team...

Reply:
Should be great to test it on dev with more than one joined team to check if selecting as default